src/main/java/org/jenkinsci/plugins/ansible/AnsibleAdHocCommandBuilder.java

```diff
@@ -225,7 +225,6 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath ws, @Nonnull Launc
             }
         } catch (IOException ioe) {
             Util.displayIOException(ioe, listener);
-            ioe.printStackTrace(listener.fatalError(hudson.tasks.Messages.CommandInterpreter_CommandFailed()));
```

Comment:
Cannot access Message anymore. Not sure if there is a replacement or can be removed

Reply:
I've inserted the literal text when I encountered this type of mistaken access to messages provided by core.